Argo.xcworkspace/xcshareddata/Argo.xcscmblueprint

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
$10 says this is Yet Another File We Need To Ignore Because Xcode Is Fucking Ridiculous (YAFWNTIBXIFR)